src/app/global_styles.tsx

```diff
@@ -16,7 +16,7 @@ const GlobalStyle = createGlobalStyle`
         height: 100%;
         width: 100%;
         font-family: Helvetica, Arial, sans-serif;
-        overflow: auto;
+        overflow: hidden;
```

Comment:
Is this to fix the scroll bar issue? When I tried similar it didn't work but seems to be fine on yours :)

Reply:
Yep, not sure why it didn't work when you tried it